src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -433,13 +442,17 @@ impl<S: StateStore> StorageTable<S> {
             false,
         );
 
+        let prefix_serializer = self.pk_serializer.prefix(pk_prefix.len());
```

Comment:
We don't need to serialize the prefix twice. We can just use the `start_key` as the encoded_prefix in `deserialize_dist_key_position_with_column_indices`